src/stream/src/executor_v2/mview/state.rs

```diff
@@ -80,16 +81,22 @@ impl<S: StateStore> ManagedMViewState<S> {
         let mut batch = self.keyspace.state_store().start_write_batch();
         batch.reserve(self.cache.len() * self.column_ids.len());
         let mut local = batch.prefixify(&self.keyspace);
+        let hash_builder = CRC32FastBuilder {};
```

Comment:
We should decide the hash function globally? cc @fuyufjh

Reply:
If we will not introduce hash functions other than CRC32, then maybe yes.